docs/reference/mapping/runtime.asciidoc

```diff
@@ -42,6 +42,28 @@ front, and can use runtime fields to amend the mapping at any time. Using
 runtime fields allows for a smaller index and faster ingest time, which
 combined use less resources and reduce your operating costs.
 
+[discrete]
+[[runtime-performance]]
+=== Performance
+You can use <<script-fields,script fields>> to access values in `_source` and
+return calculated values based on a script valuation. Runtime fields have these
+same capabilities, but provide greater flexibility because you can query and
+aggregate on runtime fields in a search request. Script fields can only fetch
+values.
+
+Similarly, you could write a <<query-dsl-script-query,script query>> that
+filters documents in a search request based on a script. Runtime fields provide
+a very similar feature that is more flexible. You write a script to create
+field values and they are available everywhere, such as
+<<search-fields,`fields`>>, <<query-dsl, all queries>>, and
+<<search-aggregations, aggregations>>.
+
+If you move a script from the `script_fields` parameter of a search request (or
```

Comment:
If we are more explicit about what we are replacing, we can be more generic here. If you move a script from any of those sections to a runtime field definition, the performance should be about the same when the script is run against the same number of documents. How a runtime field is used determines how many documents the script runs against: for instance fetching it through the fields API would only cause the script to run for the top hits, like script_fields would do. If the runtime field is queried, the perfomance characteristics are similar to those of a script query etc. What do you think?

Reply:
++ @javanna -- I added script-based sorting and clarified this section in 89cfbdddcc88105f52e10bd111328a5c83d94263